Legion/HeadOfTheLegionBeast.cs

```diff
@@ -162,7 +162,8 @@ public void SoulsHeresy(int quant = 300)
         if (!Bot.Quests.IsUnlocked(7983))
             Circles.CirclesWar(true);
         Core.Logger($"Farming {quant} Souls of Heresy");
-        Core.RegisterQuests(7983);
+        Core.RegisterQuests(7983, 7980, 7981);
+        Core.EnsureComplete(7980,7981);
```

Comment:
RegisterQuests() method autamatically accept and complete the the quests in it's parameter  so only add Core.RegisterQuests method there also add quest names next to the method in a comment // [QuestName] [QuestID]

Reply:
ok done

Reply:
> ok done

resolve my comment next time gonna merge